packages/adapter-prisma/README.md

```diff
@@ -0,0 +1,70 @@
+<!--[meta]
+section: api
+subSection: database-adapters
+title: Prisma adapter
+[meta]-->
+
+# Prisma database adapter
+
+[![View changelog](https://img.shields.io/badge/changelogs.xyz-Explore%20Changelog-brightgreen)](https://changelogs.xyz/@keystonejs/adapter-prisma)
+
+The [Prisma](https://www.prisma.io/) adapter is a general purpose adapter which can be used to connect to a range of different database backends.
+At present, the only fully tested backend is `Postgres`, however Knex gives the potential for `MSSQL`, `MySQL`, `MariaDB`, `SQLite3`, `Oracle`, and `Amazon Redshift` to be supported.
```

Comment:
```suggestion
At present, the only fully tested backend is `Postgres`, however Prisma gives the potential for `MSSQL`, `MySQL`, `MariaDB`, `SQLite3`, `Oracle`, and `Amazon Redshift` to be supported.
```

+I think those supported databases are wrong?

Reply:
Almost the whole `README` is copy pasta from knex right now. Will fix that before we ship but probably not before we've tweaked the DX a little.